### ISSUE TITLE
[RULE CHANGE REQ.] Change rule 4.1 to be more broad and accepting of names

### ISSUE THREAD

**Issue by Kyoris** [EarthMC/EarthMC.net#20]
What I mean by this is that I wish for rule 4.1 to be more accepting of nation names; mostly modifying on the first bit:

> Nation name cannot be fictional. It has to have the same name as a current or past nation.

I would like for the rule to allow names that make certain sense, pointing to the general region, rather than having to be a nation that exists/have existed. I am fine with no completely fictional names, but fictional names that are say, a mix of smaller nation/region names, should be allowed, as long as the nation does occupy those regions. A good modified version of the rule could be:

> Nation name cannot be purely fictional. It is encouraged to name your nation the same as a current or past nation, though fictional names or real-life origin are allowed, provided the reasons are clear and the name fits said nation.

**Comment by ghost** [comment]
"though fictional names or real-life origin are allowed, provided the reasons are clear and the name fits said nation."
This is very open ended and puts all the pressure on mods deciding what is and what is not appropriate.
 "provided the reasons are clear and the name fits said nation" is very ambiguous, what kind of reasons are allowed? what makes a name "fit" a nation? what would end up happening is nation names would become very inconsistent and hard to follow
This makes it far harder for mods to enforce the naming rule as they all might have different opinions on what are reasonable "clear" reasons and what names are "fitting".

**Comment by Kyoris** [comment]
Okay, to clarify things, "clear reasons" just means that you can tell me where the name comes from like _x region plus y region_ or the like, and they aren't random (has to be close by regions/nations instead of two across the world from each other)
And for the "fitting name" bit it's basically bullet point 2 on rule 4.1 moved over here, as the created name of say two African nations won't fit in siberia. Honestly idk why I did that.

**Comment by ghost** [comment]
> Okay, to clarify things, "clear reasons" just means that you can tell me where the name comes from like _x region plus y region_ or the like, and they aren't random (has to be close by regions/nations instead of two across the world from each other)
> And for the "fitting name" bit it's basically bullet point 2 on rule 4.1 moved over here, as the created name of say two African nations won't fit in siberia. Honestly idk why I did that.

Yeah that seems fitting not bashing the rule suggestion just the open wording ! Looks good, I thought the rule already operated like this but I might be wrong.

**Comment by KarlOfDuty** [comment]
The proposed rule is too open, you'd have to refine it more in that case.

**Comment by KarlOfDuty** [comment]
@Kyoris
